src/models/query.rs

```diff
@@ -3,6 +3,8 @@ use diesel::QueryResult;
 use juniper::FieldResult;
 use models::*;
 
+use models::Connection;
```

Comment:
I'm pretty sure this isn't necessary. There's already a use which should cover it above.

Reply:
yea, it doesen't understand how to collect the `use`s